### ISSUE TITLE
EDID method: does it need interlaced kernel patch?

### ISSUE THREAD

**Issue by albmm74** [D0023R/linux_kernel_15khz#12]
Hi friend.

I have been using your patches for about 4 years, but past week I decided to try the EDID method.

I caught some custom groovyarcade EDIDs for my Ubuntu 22.04.5 installation, and I tried arcade_15.bin with a default kernel (running on X).

All seems works similar, but some people with knowledge tell me that I should have problems with interlaced resolutions if the kernel has not the interlace patch.

My GPU is Radeon R9 380, and I need add the display.core=0 flag to get image.

Could you please confirm me if i could still need the patched kernel? Really I do not find any visual difference on interlaced games.

Thanks in advice.

**Comment by D0023R** [comment]
Hi there,

Thanks for sharing your experience with the patches. The EDID method is a good approach.

As for the interlaced resolutions, the need for a patched kernel depends on whether your GPU and display setup fully support interlaced output without additional tweaks. The kernel patch for interlaced mods can help ensure proper handling of these resolutions, particularly with new AMD APU and last generation Radeon. However, if you aren't noticing any visual differences when using interlaced games, it's possible that the stock kernel is handing it adequately for your current setup.

In short, it's not essential if everything looks fine, but it's worth considering to patch the kernel if you want to ensure full compatibility (1 to 1 visual experience). having it applied does not introduce any issue.

Hope this helps. Best,

**Comment by albmm74** [comment]
Thank you very much. I found your explanation helpful.

**Comment by syboxez** [comment]
Can confirm, when using amdgpu.dc=0 on an R9 380 on a stock kernel, interlacing works totally fine and has for many years.

`amdgpu.dc=0` or `display.core=0` is needed for native analog VGA to work, otherwise, you will need an external DAC.

This patch doesn't hurt, though.

**Comment by albmm74** [comment]
Hi. `display.core=0` doesn't work for me, and I did not find this command on any kernel documentation.

Be careful with the EDIDs: on my distro I do not get image on progressive resolutions with a R7 370.

**Comment by syboxez** [comment]
> Hi. `display.core=0` doesn't work for me, and I did not find this command on any kernel documentation.
> 
> Be careful with the EDIDs: on my distro I do not get image on progressive resolutions with a R7 370.

Have you tried `amdgpu.dc=0`? That works for me on my R9 380. 

As for progressive resolutions not working, make sure you have a high enough dot clock (horizontal resolution), as many GPUs on a stock kernel will not display video below a certain dot clock. For my RX 7700S, the minimum dot clock is 20MHz, for example.

**Comment by albmm74** [comment]
> > Hi. `display.core=0` doesn't work for me, and I did not find this command on any kernel documentation.
> > Be careful with the EDIDs: on my distro I do not get image on progressive resolutions with a R7 370.
> 
> Have you tried `amdgpu.dc=0`? That works for me on my R9 380.

Yes. I do.

> 
> As for progressive resolutions not working, make sure you have a high enough dot clock (horizontal resolution), as many GPUs on a stock kernel will not display video below a certain dot clock. For my RX 7700S, the minimum dot clock is 20MHz, for example.

I set a minimun dotclock of 5 on switchres for R9 380, and that value works fine for R7 370 too with patched kernel. May be some change with de EDID, I should try to increase it.

Thanks.
